### PR TITLE
Add Sidebar for Navigation

### PR DESCRIPTION
**Pull request by mivdittamo**
Add a sidebar to each page with links to navigate to any of the other pages

### CODE REVIEW

File: portfolio/src/main/webapp/aboutMe.html [hunk 1]
```diff
@@ -2,15 +2,24 @@
...
-    <img src="images/googleGLogo.jpg" width="200"/>
-    <p>I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science with concentrations in Information Internetworks and Intelligence.</p>
-    <img src="images/gtLogoBuzz.png" width="200"/>
-    <p>At school, I am a Teaching Assistant in the College of Computing! I have previously TA'ed for the Introduction to Computing course.</p>
+    <div id="content">
+        <h1>About Me</h1>
+        <p>I am Isabelle Dittamo!</p>
+        <p>I am an intern at Google as a part of the STEP Internship for the Summer of 2020!</p>
+        <img src="images/googleGLogo.jpg" width="200"/>
+        <p>I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science with concentrations in Information Internetworks and Intelligence.</p>
```
Comment (estep-google) [inline portfolio/src/main/webapp/aboutMe.html]: Generally you should character limit each line. Say 100 characters? This helps readability for the reviewer and subsequent readers of your code. Please do this here and elsewhere.

Unfortunately, I couldn't find a way to enable a suggestion line in Cloud Shell (this is something most IDEs offer).

When you do this, use the format:

```
<p>
  Text ....
  More text ...
</p>
```

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +7,28 @@
...
+#sidebar {
+    height: 100%;
+    width: 200px;
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    padding-top: 20px;
+    background-color:black;
+}
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css]: CSS best practice is to alphabetically order properties.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -7,3 +7,28 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#sidebar {
+    height: 100%;
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css]: nit: Please match indent as set in earlier definitions. Here and below.

**Review by estep-google — COMMENTED** [review]
These requests are all code style in nature.

File: portfolio/src/main/webapp/aboutMe.html (hunk 1 above)
Comment (mivdittamo) [reply portfolio/src/main/webapp/aboutMe.html]: Got it! Making those changes right now.

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (mivdittamo) [reply portfolio/src/main/webapp/style.css]: Got it! Reordering these right now.

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (mivdittamo) [reply portfolio/src/main/webapp/style.css]: Fixed indentations for style.css and went ahead and edited indentation inconsistencies in the html files.

File: portfolio/src/main/webapp/aboutMe.html [hunk 4]
```diff
@@ -9,13 +9,32 @@
...
+        <img src="images/wickedCard.jpg" height="175"/> <img src="images/LesMiserables.png" height="175"/>
+        <p>I also love traveling around the world! Here's some snapshots of some of my previous travels around Europe.</p>
+        <img src="images/spainJumpingPic.JPG" height="225"/> <img src="images/buckinghamPalace.JPG" height="225"/>
+        <img src="images/gretnaGreenHands.JPG" width="200"/> <img src="images/scotlandCastlePic.JPG" width="200"/>
+        <img src="images/irelandCastlePic.JPG" width="200"/>
+        <p>Most of all though, I love my two dogs Gumbo and Rocky!</p>
+        <img src="images/gumboGT.png" width="200"/> <img src="images/rockyStairs.JPG" width="265"/>
     </div>
-
+
```
Comment (estep-google) [inline portfolio/src/main/webapp/aboutMe.html]: Remove the whitespace from this line

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+      <meta charset="UTF-8">
+      <title>My Portfolio</title>
+      <link rel="stylesheet" href="style.css">
+      <script src="script.js"></script>
```
Comment (estep-google) [inline portfolio/src/main/webapp/index.html]: I think you went from 2 spaces to 4 spaces, but lower down you went from 4 to 2. Personally, I'm used to 2 spaces.

**Review by estep-google — COMMENTED** [review]
I think what's left is to decide on 2 spaces or 4 spaces for indent.

I would prefer 2 spaces, but you just need to keep it consistent everywhere.

File: portfolio/src/main/webapp/aboutMe.html (hunk 4 above)
Comment (mivdittamo) [reply portfolio/src/main/webapp/aboutMe.html]: Got it!

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (mivdittamo) [reply portfolio/src/main/webapp/index.html]: Changed it to make it 2 spaces for indentations.

**Review by estep-google — APPROVED** [review]
lgtm

**Pull request merged**